app/serializers/auction_with_bid_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class AuctionWithBidSerializer
+  def self.serialize(auction_with_bid)
+    auction = auction_with_bid.auction.attributes
+
+    if auction_with_bid.bid
```

Comment:
This should call the new method on the Auction model to find if there are any bids, then merge if present